### PR TITLE
Log Prompt to W&B

### PR DESCRIPTION
**Pull request by dylanuys**

### CODE REVIEW

**Review by aliang322 — COMMENTED** [review]
Looks good to merge

**Review by aliang322 — APPROVED** [review]
Looks good

File: bitmind/validator/forward.py [hunk 1]
```diff
@@ -53,33 +53,48 @@ async def forward(self):
         self (:obj:`bittensor.neuron.Neuron`): The neuron object which contains all the necessary state for the validator.
 
     """
-    #print(f"k={self.config.neuron.sample_size}")
+    wandb_data = {}
```
Comment (kenobijon) [inline bitmind/validator/forward.py:56]: where was this initiated previously?

**Review by kenobijon — APPROVED** [review]
talked offline and addressed comments about refactor

Approved

**Pull request merged**